src/test/java/posters/tests/smoke/GuestOrderTest.java

```diff
@@ -85,24 +76,24 @@ public void testOrderingAsGuest()
         final Address shippingAddress = new Address(name, company, street, city, state, zip, country);
```

Comment:
`Address` can be changed to `var`

Reply:
Please change the way that the address to be retrieved. Adapt the test case to use `DataUtils.get(Address.class)`. Please see if you can find similar places where POJOs would increase the readability of the test case.